cookbooks/bcpc/templates/default/carbon.conf.erb

```diff
@@ -284,3 +412,11 @@ MAX_AGGREGATION_INTERVALS = 5
 # seconds. Set CARBON_METRIC_INTERVAL to 0 to disable instrumentation
 # CARBON_METRIC_PREFIX = carbon
 # CARBON_METRIC_INTERVAL = 60
+
+# In order to turn off logging of successful connections for the line
+# receiver, set this to False
+# LOG_LISTENER_CONN_SUCCESS = True
```

Comment:
Not unless we have a lot of connection events, but I did not see a need to log this anyway